app/pb_kits/playbook/pb_user_badge/badges/_veteran.html.erb

```diff
@@ -0,0 +1,8 @@
+<svg viewBox="0 0 200 250" preserveAspectRatio="xMidYMid meet">
```

Comment:
`viewBox` here is very important along with `preserveAspectRatio` and [here is why](https://css-tricks.com/scale-svg/#article-header-id-1)